src/helix-ui/elements/_hx-search-bar.less

```diff
@@ -0,0 +1,58 @@
+@import './_hx-element.less';
+
+:host {
+    &([error]) {
+        #search-input {
+            border: 2px solid @red-900;
```

Comment:
Can't these styles be applied to `<hx-search>` instead of the input in the shadow dom?

Reply:
I don't see a way since `#search-input` is in the shadow dom it doesn't work when I move this to `core/hx-search`